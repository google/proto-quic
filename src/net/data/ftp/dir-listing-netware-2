total 0
- [RWCEAFMS] AK101850                         1328 Dec 27  2007 rootcert.der
d [RWCEAFMS] Admin                             512 Nov 13 07:51 Driver
d [RWCEAFMS] AK101850                          512 Nov 16 15:40 temp
