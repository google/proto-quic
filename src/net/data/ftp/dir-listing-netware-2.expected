-
rootcert.der
1328
2007
12
27
0
0

d
Driver
-1
1994
11
13
7
51

d
temp
-1
1993
11
16
15
40
